src/wolfio.c

```diff
@@ -703,8 +723,22 @@ int wolfIO_Recv(SOCKET_T sd, char *buf, int sz, int rdFlags)
 int wolfIO_Send(SOCKET_T sd, char *buf, int sz, int wrFlags)
 {
     int sent;
-
+#ifdef FUSION_RTOS
+    int err;
+retry:
+    sent = (int)SEND_FUNCTION(sd, buf, sz, wrFlags, &err);
+    if(sent < 0) {
+       if(err==63)
```

Comment:
Same here ... avoid hard coded not descriptive 63...

Reply:
Removed (customer ultimately used a custom I/O callback these changes were not necessary in the porting commit.